.github/workflows/release-published.yml

```diff
@@ -222,4 +222,47 @@ jobs:
         run: mvn -B clean deploy -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          TARGET_ARTIFACT_REPOSITORY: github
\ No newline at end of file
+          TARGET_ARTIFACT_REPOSITORY: github
+
+  deploy_xsd:
+      name: Upload xsds
+      needs: [ setup ]
+      runs-on: ubuntu-20.04
+      outputs:
+          tag: ${{ steps.collect-data.outputs.tag }}
+          version: ${{ needs.setup.outputs.version }}
+      steps:
+      - name: Download liquibase xsd
+        uses: actions/checkout@v3
+        with:
+          # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-core  
+          repository: "liquibase/liquibase"
+    
+      - name: Download liquibase-pro xsd
+        uses: actions/checkout@v3
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+        # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-pro
+          repository: "liquibase/liquibase-pro"
+
+      - name: Get major.minor xsd
+        run: |
+          version=${{ needs.setup.outputs.version }}
+          arr=(${version//./ })
+          xsd_version=${arr[0]}"."${arr[1]}
+          patch_version=${arr[2]}
+          echo ::set-env name=XSD_VERSION::${xsd_version}
+          echo ::set-env name=PATCH_VERSION::${patch_version}
```

Comment:
`::set-env` was deprecated: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
You can use `echo "YOUR_VAR=value" >> $GITHUB_ENV` and then access it like `${{ YOUR_VAR }}`